pkg/repository/v1_repository.go

```diff
@@ -14,7 +14,10 @@
 package repository
 
 import (
+	"bytes"
 	"fmt"
+	"github.com/pingcap-incubator/tiup/pkg/utils"
```

Comment:
Please fix the imports.

Reply:
Gah, sorry, my IDE just sticks them in anywhere and then hides the imports so I don't notice :-(